Google.PowerShell/ReleaseFiles/GoogleCloudPowerShell.psd1

```diff
@@ -13,7 +13,7 @@
 @{
 
 # Script module or binary module file associated with this manifest.
-RootModule = '..\PowerShell\GoogleCloud\Google.PowerShell.dll'
```

Comment:
Is changing this going to break our "add to PSModulePath" logic of the Cloud SDK installer? I don't believe so, because we add the root, "PowerShell\" folder. However, we absolutely need to confirm/test this or else we break things.